### PR TITLE
commited text with typo

### PR DESCRIPTION
**Pull request by ashutoshmimani**

### CODE REVIEW

File: temp [hunk 1]
```diff
@@ -1,2 +1,3 @@
 this is a temporary file.
 edited in Branch1
+edited agin in Branch1
```
Comment (poonammimani) [inline temp]: what is this?

Comment (ashutoshmimani) [inline temp]: komdi ka piss

Comment (poonammimani) [inline temp]: please change this

File: temp [hunk 2]
```diff
@@ -1,2 +1,3 @@
 this is a temporary file.
```
Comment (ashutoshmimani) [inline temp]: what about 1st line?

**Pull request merged**